plugins/explore/src/components/OrganizationExplorerContent/OrganizationDiagram.tsx

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Entity,
+  RELATION_CHILD_OF,
+  stringifyEntityRef,
+  ENTITY_DEFAULT_NAMESPACE,
+  parseEntityRef,
+} from '@backstage/catalog-model';
+import {
+  catalogApiRef,
+  entityRouteRef,
+  getEntityRelations,
+} from '@backstage/plugin-catalog-react';
+import {
+  DependencyGraph,
+  DependencyGraphTypes,
+  Progress,
+  useApi,
+  ResponseErrorPanel,
+  Link,
+  useRouteRef,
+  configApiRef,
+} from '@backstage/core';
+import { makeStyles, Typography } from '@material-ui/core';
+import ZoomOutMap from '@material-ui/icons/ZoomOutMap';
+import React from 'react';
+import { useAsync } from 'react-use';
+import { BackstageTheme } from '@backstage/theme';
+
+const useStyles = makeStyles((theme: BackstageTheme) => ({
+  organizationNode: {
+    fill: 'coral',
+    stroke: theme.palette.border,
+  },
+  groupNode: {
+    fill: 'yellowgreen',
+    stroke: theme.palette.border,
+  },
+}));
+
+// Simplifies the diagram output by hiding the default namespace and kind
+function readableEntityName(
+  ref:
```

Comment:
These are actually always entities right? Could use `getEntityName(ref).name` instead or at least use that or `ref.name` depending on the shape of `ref`